ui/components/src/LineChart/LineChart.tsx

```diff
@@ -194,7 +210,7 @@ export function LineChart({ height, data, unit, grid, legend, visualMap, onDataZ
     };
 
     return option;
-  }, [data, grid, legend, visualMap]);
+  }, [data, yAxis, grid, legend, visualMap]);
```

Comment:
I'm planning to refactor this in the future since the number of dependencies has grown, I also might remove the ability to customize [visualMap](https://echarts.apache.org/en/option.html#visualMap) since we didn't end up using it internally